.github/workflows/Security Vulnerability Check on Release Tags.yml

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Security Vulnerability Check on Release Tags
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  
+  pull_request:
+    branches: [main] 
+   
+env:
+  EDFI_ODS_IMP_TOKEN: ${{ secrets.REPO_DISPATCH_TOKEN }}
+  REPOSITORY_OWNER: ${{ GITHUB.REPOSITORY_OWNER }}   
+jobs:
+  Security-Vulnerability-Check-Each-Release-Tag:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag: ["v5.3-patch5","v6.1-patch3","v7.1-patch1"]   
+    name: ${{ matrix.tag }} Security-Vulnerabilities Check
+    outputs:
+          tag-status: ${{ steps.check-vulnerabilities.outputs.tag-status }}   
+    continue-on-error: true
+    steps:
+    - name: Checkout Ed-Fi-ODS
+      uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
+      with:
+        repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+        path: Ed-Fi-ODS/    
+        ref: ${{ matrix.tag }}
+    - name: Make Dependency-Check-HTML-report-${{ matrix.tag }} folder
+      run: |
+        Set-Location $env:GITHUB_WORKSPACE
+        New-Item -ItemType Directory -Path ("Dependency-Check-HTML-report-" + "${{ matrix.tag }}") -Force 
+      shell: pwsh                    
+    - name: Run dependency-check
+      uses: dependency-check/Dependency-Check_Action@main
+      with:
+          project: 'Ed-Fi-ODS'
+          path: './Ed-Fi-ODS/Application/'
+          format: 'HTML'
```

Comment:
Consider using Safire instead of HTML; I think it integrates seamlessly with GitHub actions, and we would see the results directly in the workflow results without needing to download the artifact.

Is my understanding correct @simpat-jesus?